app/models/pillar.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# Pillar represents a pillar value on Salt.
```

Comment:
Every setting that will be part of the bootstrapping process and that affects the whole cluster will be set as pillar information. E.g: the number of total etcd instances that will be created, certificate information (company name, e-mail, country...), possibly the timezone...